tests/images.rs

```diff
@@ -322,3 +329,45 @@ pub fn free_local_port() -> Option<u16> {
         .map(|addr| addr.port())
         .ok()
 }
+
+#[test]
+fn zookeeper_check_connection() {
```

Comment:
This test is doing quite a lot. Any chance we can trim does down to the very essentials that are required to assert that things are working?